internal/clientconn/conninfo/conn_info_test.go

```diff
@@ -72,3 +77,39 @@ func TestConnInfo(t *testing.T) {
 		})
 	}
 }
+
+func TestConnInfoCursorParallelWork(t *testing.T) {
```

Comment:
Does the test name make sense? I don't see it checks parallel work

Reply:
Fixed.